service/deployer/deployer.go

```diff
@@ -115,22 +119,16 @@ func (s *standardDeployer) Boot() {
 	}
 
 	for deploymentEvent := range deploymentEventChannel {
-		s.logger.Log("debug", "installing chart", "name", deploymentEvent.Name)
```

Comment:
We handle this logging, and the following two log lines, inside the `Installer` now.